src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+import { ListOfTabs } from '../../types/Tabs';
+
+type Props = {
+  tabs: ListOfTabs;
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const changeTab = (tab: Tab) => {
```

Comment:
Imho, better to use 'handleTabClick' instead of 'changeTab' .

Reply:
agree, handleTabClick sounds better